x-pack/scripts/functional_tests.js

```diff
@@ -43,6 +43,7 @@ const onlyNotInCoverageTests = [
   require.resolve('../test/licensing_plugin/config.ts'),
   require.resolve('../test/licensing_plugin/config.public.ts'),
   require.resolve('../test/licensing_plugin/config.legacy.ts'),
+  require.resolve('../test/functional_endpoint_ingest_failure/config.ts'),
```

Comment:
@oatkiller @kqualters-elastic @jfsiii Any ideas if this needs to be here when we add new testing directories? @jfsiii I didn't see the `epm_api_integration` tests in here.

Reply:
@jonathan-buttner I'm not sure. Perhaps someone from @elastic/kibana-platform or @elastic/kibana-operations can help?

Reply:
This script is currently run with the `ciGroup` tags to determine which tests will run on CI, as you can see in https://github.com/elastic/kibana/blob/f84773faed53cf0c3683406d5eaccb5f51975cda/test/scripts/jenkins_xpack_ci_group.sh#L8-L12, so yes. If you add a new FTR configuration it needs to be added to this list to be executed in CI.

Reply:
ah thanks @spalger !

Reply:
I think there was a little confusion here.  I think we want the endpoint functional UI tests in these 2 paths;
```
  require.resolve('../test/functional_endpoint_ingest_failure/config.ts'),
  require.resolve('../test/functional_endpoint/config.ts'),
```
to be in the `alwaysImportedTests` list, and not in the `onlyNotInCoverageTests` list.  If they are functional UI tests, we do want to run them in the Code Coverage job.
@wayneseymour is testing a change to move them to the other list...